app/services/clients/connector.client.js

```diff
@@ -132,6 +132,25 @@ ConnectorClient.prototype = {
     return response.data
   },
 
+  /**
+   *
+   * @param {String} serviceExternalId
+   * @param {String} accountType
+   * @param {String} credentialsId
+   * @param {Object} payload
+   * @returns {Promise}
```

Comment:
i promise that i return something

Reply:
it's async therefore it returns a promise, the return type should probably be `Promise<Object>`